action/sealedenvelope.go

```diff
@@ -0,0 +1,69 @@
+package action
+
+import (
+	"errors"
+
+	"github.com/iotexproject/go-pkgs/crypto"
+	"github.com/iotexproject/iotex-proto/golang/iotextypes"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/iotexproject/go-pkgs/hash"
+	"github.com/iotexproject/iotex-core/pkg/util/byteutil"
+)
+
+// Envelope defines an envelope wrapped on action with some envelope metadata.
```

Comment:
wrong comment. Isn't it copied from action.go?

Reply:
Repaired